WARNING  conftest:conftest.py:36 M       tests/api/tests_ui/multiconfig_config/test_environment_visible.py
INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 156
INFO     tests.api.support.base_api:base_api.py:52 Generating Jenkins API token at: "/generateNewToken"
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken HTTP/1.1" 200 168
INFO     tests.api.support.base_api:base_api.py:63 Generated token: 11bf...
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=My_Multi_Project HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /createItem?name=My_Multi_Project] Status: 200 - OK
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=Test_Folder HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /createItem?name=Test_Folder] Status: 200 - OK
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/Test_Folder/createItem?name=Pipeline_one HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /job/Test_Folder/createItem?name=Pipeline_one] Status: 200 - OK
DEBUG    conftest:conftest.py:90 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:90 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:90 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:90 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Sending stats to Plausible: Props { browser: "chrome", browser_version: "", os: "linux", arch: "x86_64", lang: "python", selenium_version: "4.31" }
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 138.0.7204.183 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 138.0.7204.183 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 138.0.7204.183
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Discovering versions from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 138.0.7204.183
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 2826 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:53055
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.08B7A1D785B26F7BE875DE1C2A73011A.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.08B7A1D785B26F7BE875DE1C2A73011A.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.08B7A1D785B26F7BE875DE1C2A73011A.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 2204f3903d46380e4fc6c9b71b41c2aeb220a4d6e1e47e825c4c17eaa45ea0c8
DEBUG    LoginPage:login_page.py:32 login crumb: 2204f3903d46380e4fc6c9b71b41c2aeb220a4d6e1e47e825c4c17eaa45ea0c8
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.B0E95AB4DCBCC4B78FFF68D6CE2C1FE4.e.70/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.B0E95AB4DCBCC4B78FFF68D6CE2C1FE4.e.71/text HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:126 Navigating to FolderPage with click on (('link text', 'Test_Folder'), 'Test_Folder')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Test_Folder'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.B0E95AB4DCBCC4B78FFF68D6CE2C1FE4.e.71/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.B0E95AB4DCBCC4B78FFF68D6CE2C1FE4.e.71/click HTTP/1.1" 200 0
DEBUG    FolderPage:base_page.py:23 Waiting for page content to load: http://localhost:8080/job/Test_Folder/
DEBUG    FolderPage:ui_element.py:28 waiting for presence_of_element_located(('css selector', 'a[href*="newJob"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/element HTTP/1.1" 200 0
DEBUG    FolderPage:base_page.py:26 Page content loaded: http://localhost:8080/job/Test_Folder/
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "POST /session/8111a870062f84101a3be6bdc7da9006/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "GET /session/8111a870062f84101a3be6bdc7da9006/element/f.7CF4768AC0E0A65313AC1AB9BB8C959B.d.50A8CD04347F35878040F076D92D8DC3.e.112/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:53055 "DELETE /session/8111a870062f84101a3be6bdc7da9006 HTTP/1.1" 200 0